examples/north_sea/plot_setup.py

```diff
@@ -15,46 +18,43 @@
 axes.axis(False)
 axes.legend()
 plt.tight_layout()
-plt.savefig("north_sea_mesh.png", dpi=300)
+imgfile = "north_sea_mesh.png"
+print_output(f'Saving {imgfile}')
+plt.savefig(imgfile, dpi=300)
 
 # Plot bathymetry in a logarithmic colourmap
-P1_2d = get_functionspace(mesh2d, "CG", 1)
-bathymetry_2d = Function(P1_2d, name="Bathymetry")
-with DumbCheckpoint("north_sea_bathymetry", mode=FILE_READ) as h5:
-    h5.load(bathymetry_2d)
 fig, axes = plt.subplots(figsize=(6.4, 4.8))
 triplot(mesh2d, axes=axes, boundary_kw={"linewidth": 0.5, "edgecolor": "k"})
-bathymetry_2d.dat.data[:] = numpy.log10(bathymetry_2d.dat.data[:])
-tc = tricontourf(bathymetry_2d, axes=axes, levels=50)
-cb = fig.colorbar(tc, ax=axes)
-ticks = cb.get_ticks()
-cb.set_ticklabels([f"{10 ** t:.0f}" for t in ticks])
+norm = mcolors.LogNorm(vmin=10, vmax=3200)
+tc = tripcolor(bathymetry_2d, axes=axes, norm=norm)
```

Comment:
I took the liberty to reformulate the plotting bits

Reply:
Fine by me, thanks!